esmvalcore/_task.py

```diff
@@ -643,7 +643,7 @@ def __repr__(self):
         string = (f"{self.__class__.__name__}: {self.name}\n"
                   f"script: {self.script}\n"
                   f"settings:\n{settings_string}\n"
-                  f"{super().str()}\n")
+                  f"{self.print_ancestors()}\n")
```

Comment:
```suggestion
                  f"ancestors: {self.print_ancestors()}\n")
```

Reply:
'ancestors: ' is already added in the by the 'print_ancestors' function, this would duplicate that